plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/server/OpenShiftServerEditorSection.java

```diff
@@ -616,7 +594,7 @@ private Job loadingProjectsJob(OpenShiftServerEditorModel model, IServerWorkingC
 	private void loadResources(final Composite container, OpenShiftServerEditorModel model) {
 		IServerWorkingCopy server = input.getServer();
 
-		final Connection connection = OpenShiftServerUtils.getConnection(server);
+		OpenShiftServerUtils.getConnection(server);
```

Comment:
I don't know if this method has a side effect or not. So I kept it, I cleaned removed the unused parameter but didn't had time to check if the getConnection call has side effect (creating the connection if not yet available for instance)